.env.sample

```
@@ -217,4 +217,25 @@ PROTOCOL=http
 # VERIFIER_CLIENT_ALIAS=VERIFIER
 # VERIFIER_DOMAIN=https://VERIFIER-domain.com
 # VERIFIER_KEYCLOAK_MANAGEMENT_CLIENT_ID=encryptedKeyCloakClientId
-# VERIFIER_KEYCLOAK_MANAGEMENT_CLIENT_SECRET=encryptedKeyCloakClientSecret
\ No newline at end of file
+# VERIFIER_KEYCLOAK_MANAGEMENT_CLIENT_SECRET=encryptedKeyCloakClientSecret
+
+# Active email provider: choose one of [resend | sendgrid | ses | smtp]
+EMAIL_PROVIDER=resend
+
+# Your Resend API key. This is required if EMAIL_PROVIDER=resend
+RESEND_API_KEY=re_xxxxxxxxxx    
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)